integration/benchmark/cmd/ci/main.go

```diff
@@ -281,3 +282,18 @@ func sendDataToBigQuery(
 	}
 	return nil
 }
+
+func outputLocalData(slices []dataSlice) {
+	jsonText, err := json.MarshalIndent(slices, "", "    ")
+	if err != nil {
+		println("Error converting slice data to json")
```

Comment:
this is going to silently ignore the json marshaling error - can we use `log.Fatal()` instead?

Reply:
Updated